src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -68,9 +69,8 @@ const TablesUserList: React.FunctionComponent<PropsType> = ({
   };
 
   const { user, profile } = useUser();
-  const { users, experience, usersById } = useSelector((state) => ({
-    users: state.firestore.ordered.partygoers,
-    usersById: state.firestore.data.users,
+  const partygoers = usePartygoers();
+  const { experience } = useSelector((state) => ({
     experience:
       state.firestore.data.experiences &&
       state.firestore.data.experiences[venueName],
```

Comment:
I think this task is not related to this PR

Reply:
Sure, I guess not directly. It is related to performance issues though. Often these things only become visible in 'unrelated PRs' when nearby code is changed, and when we do become aware of them, I want to make sure that it is captured/fixed. If not in this PR, then backlogged so we don't lose the knowledge of it once the PR it is raised in is merged.

----

While I can do it, in general I find it quicker if the PR-owner that is rejecting the suggested change/fix as 'out of scope' for this PR backlogs it at the same time. That way we reduce the time needed for back and forth in review, etc. The way I tend to do this is:

You can use 'Reference in new issue' from the `...` menu next to any comment:

![image](https://user-images.githubusercontent.com/753891/103728874-40769700-5033-11eb-8ccd-b826ef6b2d92.png)

Select the `internal-sparkle-issues` repo, then type change the `Title` to something more useful:
![image](https://user-images.githubusercontent.com/753891/103729057-a4995b00-5033-11eb-9b84-cf58d8af4098.png)

If you hold 'command' (or the equivalent 'open in new window' button for your computer) when clicking the `create issue` button, then it will create it in a new window.

You can then copy the URL, and paste it back into the comment thread so that the reviewer can see that it has been successfully backlogged.

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/117